src/components/PostBigGridItem/PostBigGridItem.js

```diff
@@ -37,6 +38,7 @@ export default function PostBigGridItem (props) {
   // XXX: we should figure out what to actually do with 'video' type attachments, which are almost never used
   let attachmentType = (firstAttachment.type === 'video' ? 'file' : firstAttachment.type) || 0
   const attachmentUrl = firstAttachment.url || 0
+  const isFlagged = post.flaggedGroups && post.flaggedGroups.includes(currentGroupId)
```

Comment:
you know, I actually would prefer this be called flaggedInGroups or groupsFlaggedIn, since its not the groups that are flagged, but the posts that are flagged. but not going to make you change it everywhere, this is good enough